src/TaskRunner/Commands/DumpCommands.php

```diff
@@ -169,20 +169,23 @@ public function toolkitCreateDump(ConsoleIO $io, array $options = [
      *
      * @option is-admin For nextcloud admin user.
      * @option yes      Skip the question to download newer dump.
+     * @option silent   Disable download progress output.
      *
      * @aliases tk-ddump
      */
     public function toolkitDownloadDump(ConsoleIO $io, array $options = [
         'is-admin' => InputOption::VALUE_NONE,
         'yes' => InputOption::VALUE_NONE,
+        'silent' => InputOption::VALUE_NONE,
```

Comment:
Instead of parameters to avoid go in a new release on CI to adjust the parameter, lets create a new config with value false and adjust wget acordingly. (see --quiet option).

For clients, in bthe case they want enable the download progress display, they will be able to do it in the runner.yml.